### PR TITLE
Enable https by default

### PR DESCRIPTION
**Pull request by ninjadq**
1. Umcomment https related configs
2. Remove the https prepare related thing in ci

Signed-off-by: DQ <dengq@vmware.com>

### CODE REVIEW

File: make/harbor.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ http:
...
-#   # https port for harbor, default is 443
-#   port: 443
-#   # The path of cert and key files for nginx
-#   certificate: /your/certificate/path
-#   private_key: /your/private/key/path
+https:
+  # https port for harbor, default is 443
+  port: 443
+  # The path of cert and key files for nginx
+  certificate: /data/cert/server.crt
```
Comment (reasonerjt) [inline make/harbor.yml]: Why not keep it as is?

**Comment by reasonerjt** [comment]
I understand the http will be redirected, but I'd suggest that we comment that section.

**Comment by reasonerjt** [comment]
Can we handle the upgrade correctly with this change?

File: make/harbor.yml (hunk 1 above)
Comment (wy65701436) [reply make/harbor.yml]: +1 

or

`/PATH_TO_YOUR_CERTIFICATE`
`/PATH_TO_YOUR_PRIVATEKEY`

Comment (wy65701436) [reply make/harbor.yml]: But, if we change back, do you need to add an installation check in the install.sh?

And, also, we need to update the travis and nightly accordingly?

**Comment by reasonerjt** [comment]
Closing as we won't ship this change in 1.9.x

**Pull request closed without merging**